### DISCUSSION TITLE
posts/blogerror/

### DISCUSSION THREAD

**Discussion by giscus** [nicejmp1/nicejmp1.github.io#16] [Comments]
# posts/blogerror/

github블로그 에러 고치기

https://nicejmp1.github.io/posts/blogerror/

**Comment by Kingsong97** [comment]
와 진짜 개열받았는데 에러때문에 감사하비다.

**Comment by KIMJW04** [comment]
글 항상 . 잘보고 있습니다 ^@^

**Comment by hyunminkim93** [comment]
잘보고 갑니다 ㅎㅎ
